src/index.tsx

```diff
@@ -823,6 +825,7 @@ export class Modalize<FlatListItem = any, SectionListItem = any> extends React.C
             )}
 
             {this.renderOverlay()}
+            {menuOverlay && menuOverlay()}
```

Comment:
Can you create a function similar to that, to keep consistency and separated? https://github.com/jeremybarbet/react-native-modalize/blob/9845fea83c0abf21a7b15b98da19ab8a6626b5d2/src/index.tsx#L737-L745